app/javascript/plugins/email_pension/EmailPensionView.js

```diff
@@ -248,7 +253,20 @@ class EmailPensionView extends Component {
                   }
                   value={this.props.email}
                   errorMessage={this.state.errors.email}
-                  onChange={value => this.props.changeEmail(value)}
+                  onChange={value => {
```

Comment:
Can this be moved to a separate function?

Reply:
Moved ✅